components/badge/badge.css

```diff
@@ -10,7 +10,6 @@
 
 .badge p {
     position: absolute;
-    background-color: var(--primary-color);
     top: -1.85rem;
     right: -0.5rem;
```

Comment:
As you can see here, you can get desired font size directly if you use px.

Reply:
I kept it as both were for the same badge